test/io/Jamfile

```diff
@@ -13,5 +13,6 @@
 # Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
 # http://www.boost.org/LICENSE_1_0.txt)
 
-build-project wkt ; 
+build-project dsv ;
```

Comment:
The `dsv` Jamfile was there, but not run. For CMake it runs without problems - so I added it for Jam too